go/cmd/nomos-wait/main.go

```diff
@@ -0,0 +1,62 @@
+// nomos-wait is a simple tool to wait until nomos has been sync'd to the current commit.
```

Comment:
What is noms and what is it used for?

Reply:
nomos is Google's GitOps tooling
https://cloud.google.com/anthos-config-management/docs

We need to wait for the actual K8s resources to be sync'd from the Git repo to the cluster

Reply:
Thanks!